### ISSUE TITLE
Ping ACK causes canzero firmware update to fail (version 0.3.2)

### ISSUE THREAD

**Issue by omzlo** [omzlo/nocand#7]
From the forum: https://forum.omzlo.com/forum/nocan/ping-ack-causes-canzero-firmware-update-to-fail-version-0-3-2

I think I reported a similar issue for the 0.2.6

```
** Sending <nocan-sys-bootloader-write node=1, func=22, param=1, len=4, data=6a8bfd26> **
2021/02/10 23:26:36 DEBUG++ SEND FRAME EXT@90341601 4: 6a 8b fd 26>
2021/02/10 23:26:36 DEBUG ** Received <nocan-sys-node-ping-ack node=1, func=9, param=0, len=0, data=> **
2021/02/10 23:26:36 WARNING Firmware upload failed: Final SYS_BOOTLOADER_WRITE failed for node 1 at address=0x0, Unexpected system message nocan-sys-node-ping-ack, while expecting nocan-sys-bootloader-write-ack.
2021/02/10 23:26:36 DEBUG++ Deleting client 3 (192.168.0.17:49925), closing channel and socket
2021/02/10 23:26:36 DEBUG ** Received <nocan-sys-bootloader-write-ack node=1, func=23, param=1, len=0, data=> **
2021/02/10 23:26:36 WARNING Message of type nocan-sys-bootloader-write-ack from node N1 (7a:9d:cc:40:9d:a3:f0:a7) was not processed
```

**Comment by omzlo** [comment]
mikb wrote:
My workaround is to add a 
ping-interval = 30000
to the config.

Presumably this will also periodically cause failures, just less frequently.

**Comment by omzlo** [comment]
Oh. This is strange. Following your previous bug report, we disabled ping messages during firmware uploads. This shouldn't happen. I will check our test bench again to see if we made a mistake.

**Comment by omzlo** [comment]
@mjbmikeb2 OK. I was able to reproduce and confirm this bug. A fix will be available soon.

**Comment by omzlo** [comment]
This bug has been fixed in 0.3.3.
